loopin.go

```diff
@@ -994,3 +994,10 @@ func (s *loopInSwap) setState(state loopdb.SwapState) {
 	s.lastUpdateTime = time.Now()
 	s.state = state
 }
+
+func (s *loopInSwap) swapInfo() *SwapInfo {
+	swapInfo := s.swapKit.swapInfo()
+	swapInfo.LastHop = s.LastHop
```

Comment:
Could be potentially dangerous as the ownership of `LastHop` is shared.

Reply:
I think we should do this in `sendUpdate` where we construct the `info`.

Reply:
I must have somehow missed `sendUpdate`. Makes a lot of sense to move this there